### PR TITLE
Add android tools extension to bazel_tools

### PR DESCRIPTION
**Pull request by comius**
Without it `android_sdk_repository` doesn't work using bzlmod.

Without bzlmod this repositories are instantiated in `src/main/java/com/google/devtools/build/lib/bazel/rules/android/android_remote_tools.WORKSPACE`

### CODE REVIEW

**Comment by comius** [comment]
cc @ahumesky @ted-xie @meteorcloudy

File: src/MODULE.tools [hunk 1]
```diff
@@ -33,3 +33,7 @@ register_toolchains("@local_config_sh//:local_sh_toolchain")
 
 remote_coverage_tools_extension = use_extension("//tools/test:extensions.bzl", "remote_coverage_tools_extension")
 use_repo(remote_coverage_tools_extension, "remote_coverage_tools")
+
+android_tools_extension = use_extension("//tools/android:extensions.bzl", "android_tools_extension")
+use_repo(android_tools_extension, "android_tools")
+use_repo(android_tools_extension, "android_gmaven_r8")
```
Comment (fmeum) [inline src/MODULE.tools:39]: `use_repo` accepts arbitrarily many positional arguments, so you can just add `, "android_gmaven_r8"` to the call above.

Comment (comius) [reply src/MODULE.tools:39]: Nice, done!

**Comment by meteorcloudy** [comment]
@ted-xie has an internal CL addressing the same problem.

**Pull request closed without merging**